org-formation/800-redirects/_tasks.yaml

```diff
@@ -97,3 +97,21 @@ SchematicDevAppRedirect:
     SourceHostedZoneId: !CopyValue [!Sub '${primaryRegion}-${resourcePrefix}-sagebio-api-zone-HostedZoneId']
     # the value of the CNAME record
     TargetHostName: !CopyValue ['schematic-dev-DockerFargateStack-LoadBalancerDNS', !Ref DnTDevAccount]
+
+# redirect schematic.api.sagebionetworks.org to schematic-infra ALB
+# https://github.com/Sage-Bionetworks/schematic-infra
+SchematicProdAppRedirect:
```

Comment:
Sorry if this is picky, but could we please use a word other than 'redirect' (both here and on line 85, above)? To me "redirect" means something very specific, i.e., to return a 3xx HTTP status.   Two alternatives terms to consider are "Routing" and "Forwarding".

Reply:
I've renamed these from "Redirect" to "DnsForward"